### DISCUSSION TITLE
欢迎加入字节跳动 - Ads Infra 团队

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#92] [Announcements]
https://imageslr.com/ads-infra.html 

 Software Elon Z

**Comment by zhangxu-ai** [comment]
大佬666

**Comment by imageslr** [comment]
> @zhangxu-ai
> 大佬666

等你简历

**Comment by zhangxu-ai** [comment]
> @imageslr
> > @zhangxu-ai
> > 大佬666
> 
> 等你简历

😂没看大佬笔记之前面了字节实习，挂了，现在在小黑屋里。凭借大佬的笔记，我侥幸拿到了腾讯的offer💕

**Comment by imageslr** [comment]
@zhangxu-ai 还是自己水平到位啦，等校招开始后联系

**Comment by kingcanfish** [comment]
大佬想问一下ads infra 组内主要用的是golang 嘛 投了实习感觉要面试一轮游了呜呜

**Comment by imageslr** [comment]
> @kingcanfish
> 大佬想问一下ads infra 组内主要用的是golang 嘛 投了实习感觉要面试一轮游了呜呜

Going / C++ 都有，我们可以邮箱交流一下

**Comment by imageslr** [comment]
今年字节跳动的提前批 7.8 就开始了，想投我们部门的同学可以先把简历发给我，我也会提些优化建议～

BTW，今年我们部门更加重视对实习/校招生的投入，近期团队入职的实习生也很多，所以今年机会是最多的，有意向的同学不要错过这个机会呀。

**Comment by imageslr** [comment]
校招正式批将于 8 月 16 日开始。通过我内推，现在就可以进入面试流程，先人一步拿 offer！

**Comment by Vector6662** [comment]
> @zhangxu-ai
> > @imageslr
> > > @zhangxu-ai
> > > 大佬666
> > 
> > 等你简历
> 
> 😂没看大佬笔记之前面了字节实习，挂了，现在在小黑屋里。凭借大佬的笔记，我侥幸拿到了腾讯的offer💕

哈哈哈我也是字节实习面挂了，现在看大佬的笔记，发现这tm不就直接是题库了吗！
非常感谢大佬的分享！

**Comment by zhangxu-ai** [comment]
> @imageslr
> @zhangxu-ai 还是自己水平到位啦，等校招开始后联系

大佬，请问字节data的端智能部门如何？拿到了这个部门的白菜offer，不知道怎么样

**Comment by imageslr** [comment]
@zhangxu-ai hi，我们可以邮箱讨论下。elonzzz@163.com

**Comment by yxheartipp** [comment]
大佬，想投您部门的实习，但是项目做过mit 6.824的一个raft kv服务，项目经验上足够吗，还是需要在丰富一下项目经历？

**Comment by imageslr** [comment]
@yxheartipp 有同学没项目经历也进来了。简历整体看吧，不一定只看项目。可以投递试试，或者邮箱发我看看。

**Comment by jingxuantju** [comment]
楼主是不是今年商业化的hc很少呀，特别难进

**Comment by imageslr** [comment]
@jingxuantju 还是有 HC 的，但相比往年少一些，整体标准会提高。

**Comment by Pang-GJ** [comment]
佬，2023年暑期实习预期hc有了吗？

**Reply by imageslr** [reply]
和往年一样，一直都有。可以先把简历发我邮箱~

**Comment by ct20000901** [comment]
大佬好😂，今天刚面了ads infra的一面，感觉结果还可以，请问今年暑期实习的hc还多吗，贵部门名声在外，有点担心后面竞争激烈被刷

**Reply by imageslr** [reply]
目前刚开始面试，还好，主要看面评。

**Comment by llyxperf** [comment]
大佬太强了，想问下部门今年转正比例有估计吗？现在是否还在招呢？

**Reply by imageslr** [reply]
转正比例一直较高。岗位在就是在招，但据我了解名额不多，竞争较大。

**Reply by llyxperf** [reply]
大佬，现在还招吗~简历已发送

**Reply by imageslr** [reply]
已回复

**Comment by alien2017** [comment]
请问一下有社招hc吗，目前在阿里做电商搜索

**Reply by imageslr** [reply]
你好，有的。可以在招聘官网搜索 `Ads Infra`：https://job.toutiao.com/s/UkcnYQQ (已附内推码) 看下是否有感兴趣的岗位，也可以发我邮箱 elonzzz@163.com。

**Comment by qianye1001** [comment]
请问今年秋招是只有北京的还在招吗，杭州和上海都没有岗位吗

**Reply by imageslr** [reply]
校招是只有北京的岗位了。杭州有一个前端全栈的日常实习，上海如果官网没有就没啦。如果你很想投递，可以单独发我简历，我问下 HR。

**Comment by Mokoghost** [comment]
想问一下商业化技术的算法岗是负责什么业务的呀，提前批要面试了不知道该准备什么😢纯小白找工作

**Reply by imageslr** [reply]
算法岗很多，不同团队的业务和方向都不同，比如有的偏策略，有的偏模型。贴一下岗位链接，我具体看看。

**Reply by Mokoghost** [reply]
https://jobs.bytedance.com/campus/m/position/7254071402256681272/detail?referral_code=TWW5BJ6

**Reply by imageslr** [reply]
他这里的描述比较抽象，看起来像是 推荐系统 的算法工程师。我认知里的广告系统的算法岗位，和这个描述不太一致。给你几个关键字：CTR / CVR 模型、数据流、PID 算法、出价策略、用户体验建模、混排策略。可以参考这两个链接：
- https://jobs.bytedance.com/experienced/position/6873455671641344264/detail?recomId=97dc9160-b1a5-11ed-bbfa-fa163eac9ea9&sourceJobId=6841007663893481741
- https://www.zhihu.com/question/318229621

总的来说，有的岗位会深入模型，写 TF 代码等。有的岗位会偏向业务策略，写的是 C++。这是核心部门。

**Reply by Mokoghost** [reply]
好的，谢谢大佬，但是面试的话应该是针对我简历问，而不会问我这些具体的业务内容吧。

**Reply by imageslr** [reply]
不会，知道这些背景算是加分项，但不是很重要，校招还是看基础水平。不过你选岗位之前可以看看和自己兴趣是否匹配。

**Reply by Mokoghost** [reply]
好的，了解了，谢谢🌹

**Comment by luckyssc** [comment]
秋招通过学习大佬的笔记面试拿到了offer，tql

**Comment by NJUSTzwh** [comment]
佬，请问组里今年还招人吗？

**Reply by imageslr** [reply]
招，之后都可以通过字节跳动的官网投递（可以填写我的内推码）

**Comment by dalingmeng** [comment]
老哥还在字节么

**Reply by imageslr** [reply]
在

**Comment by plutoyty** [comment]
hello,问下大佬，广告引擎也是属于ads infra在支持么

**Reply by plutoyty** [reply]
会有业务团队也在做广告引擎么

**Reply by imageslr** [reply]
是。没有，只有这一个大中台，服务所有业务。

**Comment by sunkashing** [comment]
求问ads-infra，投放链路组现在还卷吗？

**Reply by imageslr** [reply]
看你咋定义呢。不管哪个部门都差不多，业务体量大，工作有难度，肯定是有挑战的。

**Reply by plutoyty** [reply]
组内氛围和工作强度咋样

**Reply by imageslr** [reply]
也需要具体到部门的，不好定义，每个同学感受不太一样

**Comment by imageslr** [comment]
Update：已经从ads infra转组了。本文不再更新。还在字节，欢迎交流。

**Reply by sunkashing** [reply]
能问下为什么转组吗？

**Reply by plutoyty** [reply]
+1

**Reply by imageslr** [reply]
个人职业规划原因～

**Reply by dalingmeng** [reply]
现在在做什么方向？

**Reply by learnszl** [reply]
老哥带人了吗

**Reply by imageslr** [reply]
@dalingmeng 现在还是做搜广推架构，偏离线方向
@learnszl 没带人

**Reply by FaramitaKKK** [reply]
老哥，data下面的搜索架构做业务架构研发这个属于infra研发嘛，有点分不清架构下面的业务架构和infra的区别，我看老哥的文档，在线和离线的优化是属于业务架构嘛

**Reply by FaramitaKKK** [reply]
补充，今年拿到了data搜索架构tt业务架构的offer，需要转c++，不知道这个组前景如何呀~

**Reply by imageslr** [reply]
> 老哥，data下面的搜索架构做业务架构研发这个属于infra研发嘛，有点分不清架构下面的业务架构和infra的区别，我看老哥的文档，在线和离线的优化是属于业务架构嘛

属于业务架构，也叫infra。这边有两种架构，一种是公司层面的基础架构，做存储组件、MQ、云容器等。一种是业务架构，做整个业务链路，承上启下（对上满足业务/算法需求，实现工程链路；对下串联基础架构的各种组件，可以做业务层的定制优化）。

业务架构挺好，我做的都是业务架构。AI时代写C++/Java都没区别，另外做架构的掌握C++是必备技能了。

**Reply by FaramitaKKK** [reply]
明白，感谢哥~

**Comment by Yolo-XYX** [comment]
大佬，看你更新说去搜推架构了，请问data推荐架构下面有哪些后端架构组推荐投递呀，比较核心的？做在线推荐？如何，可以说几个组吗

**Reply by imageslr** [reply]
拆分非常多，你有备选吗

**Comment by xiaoyu-ops** [comment]
秋招前夕 重温了大佬的文章 希望一切努力可以有回报 冲鸭
